### PR TITLE
add epsilon to painn

### PR DESCRIPTION
**Pull request by Stefaanhess**

### CODE REVIEW

**Review by ktschuett — APPROVED** [review]

File: src/schnetpack/representation/painn.py [hunk 1]
```diff
@@ -75,7 +78,7 @@ def forward(
         ## intra-atomic
         mu_mix = self.mu_channel_mix(mu)
         mu_V, mu_W = torch.split(mu_mix, self.n_atom_basis, dim=-1)
-        mu_Vn = torch.norm(mu_V, dim=-2, keepdim=True)
+        mu_Vn = torch.sum(mu_V**2+self.epsilon, dim=-2, keepdim=True)
```
Comment (ktschuett) [inline src/schnetpack/representation/painn.py:81]: mu_Vn = torch.sqrt(torch.sum(mu_V**2, dim=-2, keepdim=True)+self.epsilon)

**Review by ktschuett — CHANGES_REQUESTED** [review]

**Review by ktschuett — APPROVED** [review]
